src/commands/database/delete.mjs

```diff
@@ -4,18 +4,41 @@ import { FaunaError } from "fauna";
 
 import { container } from "../../cli.mjs";
 import { throwForError } from "../../lib/fauna.mjs";
+import { getSecret, retryInvalidCredsOnce } from "../../lib/fauna-client.mjs";
 
-async function deleteDatabase(argv) {
+function validate(argv) {
+  if (!argv.secret && !argv.database) {
+    throw new Error(
+      "No secret or database provided. Please use either --secret or --database.",
+    );
+  }
+  return true;
+}
```

Comment:
could extract to `database/database.mjs` and import it in each subcommand.

Reply:
Yeah good idea